lua/core/lualine/utils.lua

```diff
@@ -0,0 +1,13 @@
+local M = {}
+
+function M.validate_theme()
+  local theme = lvim.builtin.lualine.options.theme
+
+  local lualine_loader = require "lualine.utils.loader"
+  local ok, validated_theme = pcall(lualine_loader.load_theme, theme)
+  if not ok and type(validated_theme) ~= "table" then
```

Comment:
What is the second check for ? if ok is false, then validated_theme is always nil, right ?

Reply:
That is what I expected but no.
It is a function from lualine and it returns string even if ok is false

Reply:
Ah yep, I think it returns an error as second argument.
So knowing that, we can remove the second part of the condition :)

Reply:
Lol you are right.
Thank you :)